### PR TITLE
CreateAboutPage

### PR DESCRIPTION
**Pull request by stevenTCL**
1. create About Page, move the contents on Index Page to About
2. UI modification ( in About css)
3. remove “fork me on GitHub” tab on Index and About Page

### CODE REVIEW

File: app/about.html [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+
+    <!-- CSS files -->
+    <link rel="stylesheet" type="text/css" href="css/app.css">
+    <link rel="stylesheet" type="text/css" href="css/about.css">
+    <!-- Customized libraries shared by pages -->
+    <script src="js/site.js"></script>
+</head>
+<body>
+   <!--  <style>#forkongithub a{background:#000;color:#fff;text-decoration:none;font-family:arial,sans-serif;text-align:center;font-weight:bold;padding:5px 40px;font-size:1rem;line-height:2rem;position:relative;transition:0.5s;}#forkongithub a:hover{background:#c11;color:#fff;}#forkongithub a::before,#forkongithub a::after{content:"";width:100%;display:block;position:absolute;top:1px;left:0;height:1px;background:#fff;}#forkongithub a::after{bottom:1px;top:auto;}@media screen and (min-width:800px){#forkongithub{position:fixed;display:block;top:0;right:0;width:200px;overflow:hidden;height:200px;z-index:9999;}#forkongithub a{width:200px;position:absolute;top:60px;right:-60px;transform:rotate(45deg);-webkit-transform:rotate(45deg);-ms-transform:rotate(45deg);-moz-transform:rotate(45deg);-o-transform:rotate(45deg);box-shadow:4px 4px 10px rgba(0,0,0,0.8);}}</style>
```
Comment (chautsunman) [inline app/about.html]: remove this line

File: app/about.html [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+    <span id="forkongithub"><a href="https://github.com/hkpeterpeter/teamform-seed" target="_blank">Fork me on GitHub</a></span> -->
```
Comment (chautsunman) [inline app/about.html]: remove this line

File: app/index.html [hunk 3]
```diff
@@ -30,8 +30,8 @@
...
-    <span id="forkongithub"><a href="https://github.com/hkpeterpeter/teamform-seed" target="_blank">Fork me on GitHub</a></span>
+    <!-- <style>#forkongithub a{background:#000;color:#fff;text-decoration:none;font-family:arial,sans-serif;text-align:center;font-weight:bold;padding:5px 40px;font-size:1rem;line-height:2rem;position:relative;transition:0.5s;}#forkongithub a:hover{background:#c11;color:#fff;}#forkongithub a::before,#forkongithub a::after{content:"";width:100%;display:block;position:absolute;top:1px;left:0;height:1px;background:#fff;}#forkongithub a::after{bottom:1px;top:auto;}@media screen and (min-width:800px){#forkongithub{position:fixed;display:block;top:0;right:0;width:200px;overflow:hidden;height:200px;z-index:9999;}#forkongithub a{width:200px;position:absolute;top:60px;right:-60px;transform:rotate(45deg);-webkit-transform:rotate(45deg);-ms-transform:rotate(45deg);-moz-transform:rotate(45deg);-o-transform:rotate(45deg);box-shadow:4px 4px 10px rgba(0,0,0,0.8);}}</style>
```
Comment (chautsunman) [inline app/index.html]: remove this line

File: app/index.html [hunk 4]
```diff
@@ -30,8 +30,8 @@
...
+    <span id="forkongithub"><a href="https://github.com/hkpeterpeter/teamform-seed" target="_blank">Fork me on GitHub</a></span> -->
```
Comment (chautsunman) [inline app/index.html]: remove this line

File: app/about.html [hunk 5]
```diff
@@ -0,0 +1,196 @@
...
+    <!-- UI Bootstrap CDN -->
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/angular-ui-bootstrap/2.2.0/ui-bootstrap.min.js"></script>
+
+    <!-- Angular Material CDN -->
+    <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.5.8/angular-animate.min.js"></script>
+    <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.5.8/angular-aria.min.js"></script>
+    <script src="https://ajax.googleapis.com/ajax/libs/angular_material/1.1.0/angular-material.min.js"></script>
+    <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.5.8/angular-messages.min.js"></script>
+
+    <script src="js/index.js"></script>
```
Comment (chautsunman) [inline app/about.html]: create a JS file for this page (about.js)

File: app/about.html [hunk 6]
```diff
@@ -0,0 +1,196 @@
...
+                    </div>
+                </div>
+            </div>
+        </div>
+        <hr class="featurette-divider">
+
+        <!-- START THE FEATURETTES -->
+        <div class="row featurette" id="page_team">
+            <div class="col-md-12">
+                <h2 class="featurette-heading">
```
Comment (chautsunman) [inline app/about.html]: move this <h2> out of the <div class="col-md-12">

File: app/about.html [hunk 7]
```diff
@@ -0,0 +1,196 @@
...
+                    Development Team
+                    <br>
+                    <span class="text-muted">{{teamName}}</span>
+                </h2>
+                <div class="row border">
```
Comment (chautsunman) [inline app/about.html]: use `ng-repeat`

File: app/about.html [hunk 8]
```diff
@@ -0,0 +1,196 @@
...
+    <!-- Angular Material CDN -->
+    <link rel="stylesheet" href="https://ajax.googleapis.com/ajax/libs/angular_material/0.11.2/angular-material.min.css">
+
+    <!-- CSS files -->
+    <link rel="stylesheet" type="text/css" href="css/app.css">
+    <link rel="stylesheet" type="text/css" href="css/about.css">
+    <!-- Customized libraries shared by pages -->
+    <script src="js/site.js"></script>
+</head>
+<body>
```
Comment (chautsunman) [inline app/about.html]: - add an Angular module `ng-app="teamform-about-app"`
- add a `<div>` with `ng-controller="AboutCtrl">` to include all the contents

**Comment by stevenTCL** [comment]
Just updated a new commit, plz check it out.

File: app/about.html [hunk 9]
```diff
@@ -0,0 +1,214 @@
...
+    <div ng-app="teamform-about-app" ng-controller="AboutCtrl">
```
Comment (chautsunman) [inline app/about.html]: move `ng-app` to `<body>`

**Comment by stevenTCL** [comment]
fixed

**Pull request merged**